client/Makefile

```diff
@@ -7,17 +7,17 @@ dfslib			= $(dfstools)/dfslib
 dar		        = ../dus/programs/dar/source
 ldusinc	 		= ../ldus/source/include/ldus
 utils    		= ./utils
-moving_statistics 	= ./moving_statistics
+moving_statistics 	= ./utils/moving_statistics
 secp256k1               = ../secp256k1
 OS	:= $(shell uname)
 
 ifeq ($(OS), Darwin)
 	asm_src = 						\
-		sha256-mb-x86_64-mac.s				\
+		sha256-mb-x86_64-mac.s		\
```

Comment:
Not an issue, but you broke alignment for a lot of \

Reply:
Quite interesting, I just aligned them with Sublime Text.

Reply:
![image](https://user-images.githubusercontent.com/1884084/41860049-351fa136-78d0-11e8-94bf-741f2cdd38c5.png)

Please see the alignment with vi

Reply:
Evident example of different size of tabs in different systems